app-games/src/main/AndroidManifest.xml

```diff
@@ -35,6 +35,18 @@
 
         <data android:scheme="ag" />
       </intent-filter>
+
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW"/>
+        <category android:name="android.intent.category.BROWSABLE"/>
+        <category android:name="android.intent.category.DEFAULT"/>
+        <data android:scheme="http"/>
+        <data android:scheme="https"/>
+        <data android:host="*.aptoide.com"/>
+        <data android:path="/"/>
+        <data android:path="/download"/>
+        <data android:path="/app"/>
+      </intent-filter>
       <!-- ${dataPlaceholder} -->
```

Comment:
this could be removed right ?  because this is the placeholder to make the replacement for all intent filters in vanilla v9

Reply:
If we remove the path it will open every link that has aptoide host